environment-model/environment-model_shacl.ttl

```diff
@@ -1,136 +1,159 @@
 @prefix environment-model: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/environment-model/> .
+@prefix general: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/general/> .
+@prefix georeference: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/georeference/> .
 @prefix sh: <http://www.w3.org/ns/shacl#> .
 @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
-environment-model:Asset_Shape a sh:NodeShape ;
-    sh:property [ skos:example "0.1 m" ;
-            sh:datatype xsd:float ;
-            sh:description "real texture resolution in meter (max?)"@en ;
-            sh:maxCount "1"^^xsd:unsignedInt ;
-            sh:message "Validation of texture_resolution failed!"@en ;
-            sh:minCount "0"^^xsd:unsignedInt ;
-            sh:name "texture_resolution"@en ;
-            sh:path environment-model:environment-model_quality_texture_resolution ],
-        [ skos:example "Trian3DBuilder, version 7.7" ;
-            sh:datatype xsd:string ;
-            sh:description "tool for the creation of the data"@en ;
-            sh:maxCount "1"^^xsd:unsignedInt ;
-            sh:message "Validation of creation_version failed!"@en ;
-            sh:minCount "1"^^xsd:unsignedInt ;
-            sh:name "creation_version"@en ;
-            sh:path environment-model:environment-model_project_creation_version ],
-        [ skos:example "high" ;
-            sh:datatype xsd:string ;
-            sh:description "category of the level of detail (High - highest level of detail with additional object enrichment, Med - directly from data sources, with environment, Low - topological representation)."@en ;
-            sh:maxCount "1"^^xsd:unsignedInt ;
-            sh:message "Validation of detail_level failed!"@en ;
-            sh:minCount "1"^^xsd:unsignedInt ;
-            sh:name "detail_level"@en ;
-            sh:path environment-model:environment-model_quality_detail_level ],
-        [ skos:example "12000" ;
-            sh:datatype xsd:int ;
-            sh:description "Total number of all geoemtries, instances are considered only once"@en ;
-            sh:maxCount "1"^^xsd:unsignedInt ;
-            sh:message "Validation of geometry_count failed!"@en ;
-            sh:minCount "1"^^xsd:unsignedInt ;
-            sh:name "geometry_count"@en ;
-            sh:path environment-model:environment-model_quantity_geometry_count ],
-        [ skos:example "512m" ;
-            sh:datatype xsd:float ;
-            sh:description "tile size in meter"@en ;
-            sh:maxCount "1"^^xsd:unsignedInt ;
-            sh:message "Validation of tile_size failed!"@en ;
-            sh:minCount "0"^^xsd:unsignedInt ;
-            sh:name "tile_size"@en ;
-            sh:path environment-model:environment-model_quantity_tile_size ],
-        [ skos:example "DataSmith" ;
+environment-model:environmentModelShape a sh:NodeShape ;
```

Comment:
Classnames in PascalCase
--> EnvironmentModelShape

Reply:
fixed